getting-started/exploring-cloud.md

```diff
@@ -0,0 +1,119 @@
+This will be sufficient to complete all our tutorials and run a few test projects.
+
+If you're interested in learning more about pricing of Timescale Cloud, visit the
+[Timescale Cloud pricing calculator][timescale-pricing]. Or, [contact us][contact]
+and we will be happy to walk you through the best Timescale Cloud configuration
+for your use cases.
+
+Once you've selected your service options, click `Create Service`.
+
+It will take a few minutes for your service to provision in your cloud. Now is
+a good time to familiarize yourself with some of the [features of TimescaleDB][using-timescale]
+and our [getting started tutorial][hello-timescale].
+
+### Step 3: Install psql
+
+Nearly all of our tutorials require some working knowledge of `psql`, the command-line
+utility for configuring and maintaining PostgreSQL. We recommend
+[installing psql][install-psql].
+
+### Step 4: Connect to your database using psql
+
+You will see a green `Running` label and a green dot under the "Nodes" column when 
+your instance is ready for use.
+
+Once your instance is ready, navigate to the ‘Overview Tab’ of your Timescale 
+Cloud dashboard and locate your `host`, `port`, and `password`, as highlighted below.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/hello-timescale/NYC_figure1_1.png" alt="NYC Taxis"/>
+
+Afterward, connect to your Timescale Cloud database from `psql`
+by typing the command below into your terminal,
+ensuring that you replace the {curly brackets} with your real
+password, hostname, and port number found in the overview tab.
+
+```bash
+psql -x "postgres://tsdbadmin:{YOUR_PASSWORD_HERE}@{YOUR_HOSTNAME_HERE}:{YOUR_PORT_HERE}/defaultdb?sslmode=require"
+```
+
+You should see the following connection message:
+
+```bash
+psql (12.2, server 11.6)
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+Type "help" for help.
+tsdbadmin@defaultdb=>
+```
+
+### Step 5: Verify that TimescaleDB is installed
+
+To verify that TimescaleDB is installed, run the `\dx` command
+to list all installed extensions to your PostgreSQL database.
+You should see something similar to the following output:
+
+```sql
+                  List of installed extensions
+| Name        | Version | Schema     | Description                                  |
+|-------------|---------|------------|----------------------------------------------|
+| plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language                 |
+| timescaledb | 1.6.0   | public     | Enables scalable inserts and complex queries |
+```
+
+### Step 6: Hello, Timescale!
+
+Congratulations! You are now up and running with Timescale Cloud. In order to
+familiarize yourself with the features and capabilities of the product, we
+recommend that you complete the [Hello, Timescale!][hello-timescale] tutorial.
+
+[create-login]: /getting-started/installation
```

Comment:
Don't think this is what you intended.  `https://www.timescale.com/cloud-signup`, right?